src/sentry/services/hybrid_cloud/organization/model.py

```diff
@@ -98,6 +98,7 @@ class RpcOrganizationMemberFlags(RpcModel):
     member_limit__restricted: bool = False
     idp__provisioned: bool = False
     idp__role_restricted: bool = False
+    partnership__restricted: bool = False
```

Comment:
This is a new flag that is not used anywhere but will be to mark partnership org member.

Reply:
oh i was looking for this but i forgot where it was. thanks!